### ISSUE TITLE
Provided pretrained model(eu, global) structure is different to source code  

### ISSUE THREAD

**Issue by mfchen881** [ankandrew/fast-plate-ocr#29]
Hi, thanks for the great work on the license plate OCR! It’s really helpful for me.
I’m trying to train a European license plate OCR model for my own ALPR project, and I would like to fine-tune the trained model from “european_mobile_vit_v2_ocr.keras” which you provided. In your latest updates, you provided the load weight function for fine-tuning, so I tried to extract the weight file from the Keras model, but it returned an error. Below is my extract code and error message:
 ```
from fast_plate_ocr.train.model.config import load_config_from_yaml
from fast_plate_ocr.train.utilities.utils import load_keras_model
config = load_config_from_yaml(config_path)
model = load_keras_model(
    model_path,
    config.vocabulary_size,
    config.max_plate_slots
)
model.save_weights(weight_path)
```
 ![image](https://github.com/user-attachments/assets/4ac4c69b-c84d-4f03-9d14-a5e327596a9a)

My environment is on the Docker image: nvcr.io/nvidia/tensorrt:24.06-py3, and I installed the dependencies from the poetry which you provided.

Additionally, I noticed that the model you provided (EU and global) does not have the same structure as the source code when I use netron.app to visualize the model structure. The left one is the “european_mobile_vit_v2_ocr.keras” structure, and the right one is the raw model which is extracted directly from “fast_plate_ocr.train.model.models.cnn_ocr_model”.
![image](https://github.com/user-attachments/assets/d6c06011-c992-414d-905f-e48b4157b890) ![image](https://github.com/user-attachments/assets/4d60dec6-be3b-4d43-af3e-982fce291feb)

As you can see, the EU pretrained model uses zero-padding instead of max-pooling. Moreover, the activation layers are also different; while the pretrained model uses ‘silu’, the source code defaults to ‘relu’. Could you provide the new model file or proper weight files which fit the source code(both EU & global)? Thank you!

**Comment by ankandrew** [comment]
Hi! Your analysis is correct, the model in `master` branch contains the conv-only model. The european / global contains MobileVitV2 architecture, which is only present in https://github.com/ankandrew/fast-plate-ocr/compare/master...MobileViTV2-backbone that branch. I don't have time but my plans were to make it configurable the model architecture to use. Feel free to contribute if you want!

Meanwhile I would just use that branch for fine-tuning (and I'd fine tune with the `global` one, is has been fine-tuned from the `european` one but with much more plates!). The rest of the code stays the same to do inference.

**Comment by mfchen881** [comment]
Hi, thanks for your information! I didn't notice the other branches. Now I can load the model smoothly!

Regarding the crop_to_aspect_ratio error, I was using v3.2.1 of Keras. This problem is solved when I upgraded Keras to v3.7.0.

Thank you again for your help and answers!

**Comment by ankandrew** [comment]
Great that it worked!
